verilog/analysis/checkers/endif_comment_rule.cc

```diff
@@ -120,14 +120,35 @@ void EndifCommentRule::HandleToken(const TokenInfo& token) {
           auto endif_end =
               last_endif_.text().substr(last_endif_.text().size(), 0);
 
+          // The whitespace before the comment is something that should come
+          // from the formatter configuration so that we don't emit code that
+          // then would be complained about by the formatter.
+          // Or, make this a configuration option in the lint rule (probably
+          // best, as we don't intermingle formatter/linter).
+          // TODO(hzeller) Until that is happening just comment here.
+          constexpr int kSpacesBeforeComment = 2;
+          std::string ws(kSpacesBeforeComment, ' ');
+
+          // TOOD(hzeller): rethink the 'alternative fixes' thing. Problem is,
```

Comment:
@mglb The alternatives are probably hard to surface in the UI (right now, we just choose the first one), and in particular for unattended patch generation, this is also not working well.

So maybe we should just add a configuration to this lint rule (configuration infrastructure is there), to pre-choose how fixes are generated. That should simplify things. WDYT ?

Reply:
We can call this configuration option "default choice". This will work well even with (hypothetical) UIs that offer choice picking (language server or other IDE integration comes to mind).

Reply:
mmh, ok that could be a way. I'd still see that a single choice, chosen in the configuration file of the project as part of the style-guide, would simplify the workflow, as the IDE's task is to reduce decision fatigue.

Anyway, something for another change in the future.